data/adapters/gersbach_E2G_CRISPR_adapter.py

```diff
@@ -0,0 +1,119 @@
+                    element_type = 'transcription start site'
+                _id = genomic_element + '_' + self.file_accession
+                _props = {
+                    '_key': _id,
+                    'name': _id,
+                    'chr': genomic_elements[genomic_element]['chr'],
+                    'start': genomic_elements[genomic_element]['start'],
+                    'end': genomic_elements[genomic_element]['end'],
+                    'method_type': 'CRISPR',
+                    'type': element_type,
+                    'source': GersbachE2GCRISPR.SOURCE,
+                    'source_url': self.source_url,
+                    'files_filesets': 'files_filesets/' + self.file_accession
+                }
+                self.writer.write(json.dumps(_props))
+                self.writer.write('\n')
+        elif self.label == 'genomic_element_gene':
+            ffs = FileFileSet(accessions=[], writer=None,
+                              label='igvf_file_fileset')
+            file_set_props, _, _ = ffs.query_fileset_files_props_igvf(
+                self.file_accession, replace=False)
+            biosample_context = file_set_props['simple_sample_summaries']
+            biosample_term = file_set_props['samples']
+            biosample_qualifier = file_set_props['treatments_term_ids']
+            method = file_set_props['method']
+            with gzip.open(self.data_file, 'rt') as data_file:
+                reader = csv.reader(data_file, delimiter='\t')
+                next(reader)
+                for row in reader:
+                    guide_id = row[0]
+                    if guide_id not in guide_id_to_element_id:
+                        print(guide_id_to_element_id)
+                        raise ValueError(
+                            f'{guide_id} not in guide RNA sequences file.')
+                    element_id = guide_id_to_element_id[guide_id]
+                    gene = genomic_elements[element_id]['gene']
+                    log2FC = float(row[2])
+                    _id = '_'.join([element_id, gene, self.file_accession])
+                    _source = 'genomic_elements/' + element_id + '_' + self.file_accession
+                    _props = {
+                        '_key': _id,
+                        '_from': _source,
+                        '_to': 'genes/' + gene,
+                        'log2FC': log2FC,
+                        'source': GersbachE2GCRISPR.SOURCE,
+                        'source_url': self.source_url,
+                        'files_filesets': 'files_filesets/' + self.file_accession,
+                        'name': 'modulates expression of',
+                        'inverse_name': 'expression modulated by',
+                        'method': method,
+                        'biosample_context': biosample_context,
+                        'biosample_term': biosample_term,
```

Comment:
those two fields above were loaded as `simple_sample_summaries` and `biological_context` in encode_element_gene.py, but I can rename them in DSERV-851, think the names here make more sense.

Reply:
Ok.

Reply:
actually decided to go the other way, we made a new ticket (DSERV-871) to also go through other collections

Reply:
so follow what encode_element_gene.py is doing currently: add biological_context, simple_sample_summaries here

Reply:
Ok. Also I renamed treatments from biosample_qualifier to treatments_term_ids to be consistent with that adapter.